### PR TITLE
Clariry that App ID !== Client ID

### PR DESCRIPTION
**Pull request by WyriHaximus**
When implementing this I made the mistake of assuming that the Client ID would be the same as the App ID. A few frustrating hours and a support ticket later I was pointed out this important difference. This commit introduces a small note about this difference. Hopefully saving others a few hours of trying to figure this out.

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Review by 0503256405 — DISMISSED** [review]
[moderator removed content]

**Review by 0503256405 — COMMENTED** [review]
[moderator removed content]

**Review by jigar-sable — DISMISSED** [review]

**Comment by janiceilene** [comment]
Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by jaceppo — APPROVED** [review]

**Review by jaceppo — APPROVED** [review]
ok

**Comment by WyriHaximus** [comment]
> Thanks so much for opening a PR! I'll get this triaged for review ✨

Took me a few hours to be pointed out my mistake, this is the least I can do 👍

**Comment by sarahs** [comment]
👋 We did some content restructuring that resulted in a conflict with your PR. It looks like we may not have access to check out your fork. Let us know if you need a hand resolving the conflict!

**Comment by WyriHaximus** [comment]
>  We did some content restructuring that resulted in a conflict with your PR. It looks like we may not have access to check out your fork. Let us know if you need a hand resolving the conflict!
> 
> The new location of the file is: `content/developers/apps/building-github-apps/identifying-and-authorizing-users-for-github-apps.md`

Done :+1:

File: content/developers/apps/building-github-apps/identifying-and-authorizing-users-for-github-apps.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ When your GitHub App specifies a `login` parameter, it prompts users with a spec
 
 Name | Type | Description
 -----|------|------------
-`client_id` | `string` | **Required.** The client ID for your GitHub App. You can find this in your [GitHub App settings](https://github.com/settings/apps) when you select your app.
+`client_id` | `string` | **Required.** The client ID for your GitHub App. You can find this in your [GitHub App settings](https://github.com/settings/apps) when you select your app. (Note that the app ID and client ID are not the same, and are not interchangeable.)
```
Comment (jmarlena) [inline content/developers/apps/building-github-apps/identifying-and-authorizing-users-for-github-apps.md:50]:
```suggestion
`client_id` | `string` | **Required.** The client ID for your GitHub App. You can find this in your [GitHub App settings](https://github.com/settings/apps) when you select your app. **Note:** The app ID and client ID are not the same, and are not interchangeable.
```

**Review by jmarlena — APPROVED** [review]
🚢  Thanks for surfacing this tip to help users avoid confusion!! This looks good to merge once the tests all pass.

**Comment by 0503256405** [comment]
Umairkhan0503256405@gmail.com

**Pull request merged**